graph/patterns/enums.md

```diff
@@ -1,25 +1,33 @@
-### Enums
+# Enums
 
-In OData, enums represent a subset of the nominal type they rely on, and are especially useful in cases where certain properties have predefined, limited options.
+Microsoft Graph API Design Pattern
 
-```xml
-<EnumType Name="color">
-    <Member Name="Red" Value="0" />
-    <Member Name="Green" Value="1" />
-    <Member Name="Blue" Value="2" />
-</EnumType>
-```
+Enums represent a subset of the nominal type they rely on, and are especially useful in cases where certain properties have predefined, limited options.
+
+## Problem
+
+The API Design requires that a specific property of a data structure has a finite and limited set of values, and such limitation needs to be clearly communicated to the user and give the developer the same structure

```

Comment:
Could you please ground the problem in a real design scenario. A good example is at https://github.com/microsoft/api-guidelines/blob/vNext/graph/patterns/alternate-key.md#problem or https://github.com/microsoft/api-guidelines/blob/vNext/graph/patterns/evolvable-enums.md#problem.